### PR TITLE
Update drip

### PR DESCRIPTION
**Pull request by tamouse**
Update to include Drip experience.

### CODE REVIEW

File: data/resume.yml [hunk 1]
```diff
@@ -72,96 +76,93 @@ summary_links:
   - text: Technical Debt (Martin Fowler)
     link: http://martinfowler.com/bliki/TechnicalDebt.html
 brief_summary: |
-    Deeply experienced in software design and development methodologies,
-    especially related to the disciplines of software quality, web
-    applications, user experience, and team building. Presently working at the intersection of Rails, GraphQL, and React, developing modern web applications with fast response times.
+    Deeply experienced in software architecture, design, and development methodologies,
+    especially related to the disciplines of web applications, user experience, team
+    building, and processes. Working at the intersection of Rails, GraphQL, and React,
```
Comment (sharpfive) [inline data/resume.yml]: > Working at the intersection of Rails, GraphQL, and React

This stands out and gave me a clear impression of your skills. Something I can share with others.

> developing modern web applications with fast response times.

^^ same here. Maybe put these at the top of the summary?

File: data/resume.yml [hunk 2]
```diff
@@ -72,96 +76,93 @@ summary_links:
...
 methodologies:
-  - Program architecture and design
-  - User-/task-centered design
-  - Test-/behaviour-driven development
-  - Structured analysis, design, development, and testing
-  - Process flow analysis, systems theory, cybernetics
-  - Object-oriented analysis, design, programming, and testing
   - Agile development, scrum master
-  - 'Continuous everything: integration, testing, deployment'
+  - Continuous everything -- integration, testing, deployment
```
Comment (sharpfive) [inline data/resume.yml]: If I'm looking for CI/CD help, I'm probably scanning each resume for 

> CI/CD

 and might overlook how you specified it here

I liked that you separated out integration and deployment. It shows you understand they are not-the-same-thing.

**Review by sharpfive — COMMENTED** [review]
Great experience!

File: data/resume.yml (hunk 1 above)
Comment (tamouse) [reply data/resume.yml]: yes.. don't bury the lead

File: data/resume.yml (hunk 2 above)
Comment (tamouse) [reply data/resume.yml]: 👍🏻

**Comment by tamouse** [comment]
thanks, @sharpfive , great feedback

File: data/resume.yml [hunk 3]
```diff
@@ -72,96 +76,93 @@ summary_links:
...
-    - software engineering
-    - web development
     - business analysis
     - consulting
     - facilitating
+    - software engineering
     - teaching
+    - web development
   software_engineering:
+    - "APIs, REST, GraphQL"
```
Comment (thekbb) [inline data/resume.yml]: This was good to move up to the top, but how you had it before was clearer.
suggestion:
```
APIs: REST, GraphQL
```

Comment (tamouse) [reply data/resume.yml]: @thekbb : I wonder how to express both using and creating APIs in different forms...

**Comment by tamouse** [comment]
> Great experience!

"When old as I am, look as good you will not" — yoda

🤣

**Pull request merged**